docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -0,0 +1,189 @@
+# Manage XMTP inboxes
+
+The first time someone uses an installation of your app, your app creates an inbox ID and installation ID associated with their wallet address. To do this, you [create a client](https://docs.xmtp.org/inboxes/build-inbox#create-an-xmtp-client) for their wallet address. 
+
+The client creates an inbox ID and installation ID associated with the wallet address. By default, this wallet address is designated as the recovery address. A recovery address will always have the same inbox ID and cannot be reassigned to a different inbox ID. 
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/create-inbox-install.png" width="250px" />
+</div>
+
+Each time you make a subsequent call to create a client for the same wallet address, the client uses the same inbox ID, but creates a new installation ID.
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/add-installation.png" width="250px" />
+</div>
+
+You can enable a user to add multiple wallet addresses to their inbox. Added addresses use the same inbox ID and the installation ID of the installation used to add the address.
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/add-address.png" width="350px" />
+</div>
+
+You can enable a user to remove a wallet address from their inbox. You cannot remove the recovery wallet address.
+
+## Add a wallet address to an inbox
+
+:::code-group
+
+```jsx [React Native]
+await client.addAccount(walletToAdd)
+```
+
+```kotlin [Kotlin]
+client.addAccount(walletToAdd)
+```
+
+```swift [Swift]
+try await client.addAccount(newAccount: walletToAdd)
+```
+
+:::
+
+## Remove a wallet address from an inbox
+
+:::tip[Note]
+You cannot remove a recovery wallet.
+:::
+
+:::code-group
+
+```jsx [React Native]
+await client.removeAccount(recoveryWallet, addressToRemove)
+```
+
+```kotlin [Kotlin]
+client.removeAccount(recoveryWallet, addressToRemove)
+```
+
+```swift [Swift]
+try await client.removeAccount(recoveryAccount: recoveryWallet, addressToRemove: addressToRemove)
+```
+
+:::
+
+## Revoke all other installations
+
+Your user may encounter a case where a wallet address for one of their installation IDs gets compromised:
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/compromised-address.png" width="350px" />
+</div>
+
+In this case, you can revoke the address’ access to all other installations:
```

Comment:
How do we give back installation access to the wallet address?

Reply:
So right now we only expose `revokeAllOtherInstallations`. Which means you cannot revoke the current installations access only all other installations. This does not remove an address from the account it just removes installations of this inboxId from the account.